src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,77 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+    //combine these into quality changer--takes in a + or - number and you can get BOTH increase or decrease in quality
+    qualityUpdater = (item, change_Amount) =>{
```

Comment:
I think you're missing a declaration here!
```suggestion
    const qualityUpdater = (item, change_Amount) =>{
```